src/Controller.js

```diff
@@ -0,0 +1,119 @@
+const {
+  gameStart,
+  bridgeSize,
+  moving,
+  gameCommand,
+  gameMap,
+  gameResult,
+} = require("./UI/View");
+const {
+  APPROPRIATE_INPUT,
+  ANSWER,
+  REPLACEMENT,
+  GAME_MESSAGE,
+} = require("./Utils/Constants");
+const { UP, RESTART, QUIT } = APPROPRIATE_INPUT;
+const { RIGHT, WRONG, UNCHOSEN } = ANSWER;
+const { COMMA, LINE } = REPLACEMENT;
+const { SUCCESS, FAIL } = GAME_MESSAGE;
+const { close } = require("./Utils/MissionUtils");
+
+const BridgeGame = require("./BridgeGame");
+const BridgeMaker = require("./BridgeMaker");
+const BridgeRandomNumberGenerator = require("./BridgeRandomNumberGenerator");
+
+class Controller {
+  #bridge;
+  #round = 0;
```

Comment:
[eslint lines-between-class-members](https://eslint.org/docs/latest/rules/lines-between-class-members)에 의하면 아래처럼 한 줄 띄어쓰는 것을 권장한다고 합니다!
```suggestion
  #bridge;
  
  #round = 0;
```